src/core/Modal/Modal.tsx

```diff
@@ -170,10 +176,20 @@ export const Modal = (props: ModalProps) => {
 
   return !!container ? (
     ReactDOM.createPortal(
-      isOpen && (
-        <FocusTrap>
+      <FocusTrap>
```

Comment:
Move it inside `CSSTransition`

Reply:
Ah, you also need to add `focusRef` to `FocusTrap`. You can check other components, simplest example `Body` component.

Reply:
Done. Changes in CSS were needed but they are mostly complete now. I have a PR that fixes an issue I introduced with the regular modal animations.